README.md

```diff
@@ -136,290 +156,240 @@ interface RestfulProviderProps<T> {
-    <div>
-      <h1>Are you ready?</h1>
-      <p>Are you ready to unleash all the magic? If yes, click this button!</p>
-      <button onClick={get}>GET UNICORNS!!!!!!</button>
+import React from "react";
+import { useGet } from "restful-react";
 
-      {unicorns && (
-        <ul>
-          {unicorns.map((unicorn, index) => (
-            <li key={index}>{unicorn}</li>
-          ))}
-        </ul>
-      )}
+const MyComponent = () => {
+  const { data: randomDogImage, loading, refetch } = useGet({
+    path: "https://dog.ceo/api/breeds/image/random",
+    lazy: true,
+  });
+
+  return !randomDogImage && loading ? (
+    <h1>Loading!</h1>
+  ) : (
+    <div>
+      <div>
+        <h1>Welcome to my image getter!</h1>
+        <button onClick={() => refetch()}>Get a good boye!</button>
+      </div>
+      <div>{randomDogImage && <img alt="Here's a good boye!" src={randomDogImage.message} />}</div>
     </div>
-  )}
-</Get>
+  );
+};
+
+export default MyComponent;
 ```
 
-The above example will display your UI, and then load unicorns on demand.
+The above example will display your UI, and then load good boyes on demand.
 
 ### Response Resolution
 
-Sometimes, your backend responses arrive in a shape that you might want to adapt, validate, or reshape. Other times, maybe your data consistently arrives in a `{ data: {} }` shape, with `data` containing the stuff you want.
+Sometimes, your backend responses arrive in a shape that you might want to adapt, validate, or restructure. Other times, maybe your data consistently arrives in a `{ data: {} }` shape, with `data` containing the stuff you want.
 
-At the `RestfulProvider` level, _or_ on the `Get` level, a `resolve` prop will take the data and _do stuff_ to it, providing the final resolved data to the children. Consider,
+At the `RestfulProvider` level, _or_ on the `useGet` level, a `resolve` prop will take the data and _do stuff_ to it, providing the final resolved or unwrapped data to the children. Consider,
 
-[![Edit Restful React demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/30n66z45mq)
+[![Edit restful-react demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/restful-react-demos-vjets)
 
 ```jsx
-const myNestedData = props => (
-  <Get
-    path="/this-should-be-simpler"
-    resolve={response => response.data.what.omg.how.map(singleThing => singleThing.name)}
-  >
-    {data => (
-      <div>
-        <h1>Here's all the things I want</h1>
-        <ul>
-          {data.map(thing => (
-            <li>{thing}</li>
-          ))}
-        </ul>
-      </div>
-    )}
-  </Get>
-);
+import React from "react";
+import { useGet } from "restful-react";
+
+const MyComponent = () => {
+  const { data: imageUrl } = useGet({
+    path: "https://dog.ceo/api/breeds/image/random",
+    resolve: image => image && image.message,
+  });
+
+  return <img alt="Here's a good boye!" src={imageUrl} />;
```

Comment:
this will show broken image for a moment, not sure if it matters for the exampls

Reply:
So would you suggest

```suggestion
  return imageUrl && <img alt="Here's a good boye!" src={imageUrl} />;
```

?